### PR TITLE
Added AutoML team reviewers

### PR DESCRIPTION
**Pull request by harshithapv**
Added AutoML team reviewers to files owned by AutoML team

So we can do the best job, please check:

- [X ] There's a descriptive title that will make sense to other developers some time from now. 
- [ ] There's associated issues. All PR's should have issue(s) associated - unless a trivial self-evident change such as fixing a typo. You can use the format `Fixes #nnnn` in your description to cause GitHub to automatically close the issue(s) when your PR is merged.
- [ ] Your change description explains what the change does, why you chose your approach, and anything else that reviewers should know.
- [ ] You have included any necessary tests in the same PR.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,6 +2,12 @@
 # For more information on CODEOWNERS file see : https://help.github.com/en/articles/about-code-owners
 
 # For the entire repository 
-* @dotnet/mlnet-core
- 
+*	@dotnet/mlnet-core
 
+# Reviewers for files owned by AutoML team
+src\Microsoft.ML.AutoML	@dotnet/mlnet-automl
+src\mlnet	@dotnet/mlnet-automl
```
Comment (eerhardt) [inline .github/CODEOWNERS]: This folder doesn't exist.

Same for `test\mlnet.Tests`

Comment (codemzs) [reply .github/CODEOWNERS]: @harshithapv can you please remove these folders? @Dmitry-A can you please confirm if there any other folders you would want to own?

Comment (Dmitry-A) [reply .github/CODEOWNERS]: mlnet stuff is currently in a feature branch but we will be merging it sometime soon, so I thought to add them here ahead of time.  We can either do that or add them later, either way works for me.

Comment (codemzs) [reply .github/CODEOWNERS]: Lets add it after we merge. It does not make sense to add path to a folder that does not exist.

**Pull request merged**